var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -60,5 +62,7 @@ class PyPytest(PythonPackage):
     depends_on('py-pathlib2@2.2.0:', when='@3.7.1: ^python@:3.5', type=('build', 'run'))
     depends_on('py-pathlib2', when='@3.7.0 ^python@:3.5', type=('build', 'run'))
     depends_on('py-importlib-metadata@0.12:', when='@4.6:5.0', type=('build', 'run'))
-    depends_on('py-importlib-metadata@0.12:', when='@5.1: ^python@:3.7', type=('build', 'run'))
-    depends_on('py-wcwidth', when='@4.5:', type=('build', 'run'))
+    depends_on('py-importlib-metadata@0.12:', when='@5.1:5.999 ^python@:3.7.999', type=('build', 'run'))
```

Comment:
This dependency is still needed in newer versions, you can undo this change.

Reply:
done